Makefile

```diff
@@ -36,6 +36,7 @@ tools:
 build: api-server build-only
 
 lint:
+	go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.50.0
```

Comment:
We have to install the tool each time we run `make lint`, anyway to simplify it?

Reply:
it's pinned to version 1.50.0.  If it's already installed locally it which check the hash of this release against your local version and won't re-install.